### PR TITLE
feat: create initial structure for staticpub.distributed.press

### PR DESCRIPTION
**Pull request by akhileshthite**
https://github.com/orgs/hyphacoop/projects/20/views/1?pane=issue&itemId=60232438&issue=hyphacoop%7Cdistributed-press-organizing%7C130

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,46 @@
 # staticpub.distributed.press
+
 Template repo for ActivityPub enabled static websites that use the Distributed Press SDK
+
+## Instructions
+
+To replace all instances of `staticpub.mauve.moe` with `yourdomain.com`, `mauve` with `username`, and `"Mauve"` with `"Your Name"`, you can use the following sed commands:
```
Comment (RangerMauve) [inline README.md]: Change this from `staticpub.mauve.moe` to `staticpub.distributed.press`. Maybe also change all instances of `Mauve` to `dp`?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+find . -type f -exec sed -i 's/staticpub\.mauve\.moe/yourdomain\.com/g; s/mauve/username/g; s/"Mauve"/"Your Name"/g' {} +
+```
+
+### For macOS Users:
+
+```bash
+find . -type f -exec sed -i '' 's/staticpub\.mauve\.moe/yourdomain\.com/g; s/mauve/username/g; s/"Mauve"/"Your Name"/g' {} +
+```
+
+Make sure to update the `publicKeyPem` field in the following files with your actual public key from the `.dprc` configuration file:
```
Comment (RangerMauve) [inline README.md:21]: Link to the docs page for setting up the CLI

**Review by RangerMauve — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  <link rel="alternate" type="application/activity+json" href="./about.jsonld" />
+</head>
+<body>
+  <h1>Welcome to Your Static Site</h1>
+  <nav>
+    <ul>
+      <li><a href="./about.html">About</a></li>
+      <li><a href="./outbox.html">Posts</a></li>
+    </ul>
+  </nav>
```
Comment (RangerMauve) [inline index.html:15]:
```suggestion
  </nav>
  <footer>
  <p style="font-family: Source Sans Pro; font-size: 0.8em;">
    <a href="//distributed.press">Powered By Distributed Press</a>
  </p>
  </footer>
```

**Review by RangerMauve — APPROVED** [review]
I set up the DNS, see if it works after a publish.

**Comment by akhileshthite** [comment]
> I set up the DNS, see if it works after a publish.

Same issue; ipns and hyper are working but not http
Screenshot 2024-11-28 at 10 06 30 AM

**Comment by RangerMauve** [comment]
HTTPS works. Mind updating the "powered by distributed press" snippet then it should be good to merge.

**Comment by akhileshthite** [comment]
> HTTPS works. Mind updating the "powered by distributed press" snippet then it should be good to merge.

Yep, it's updated https://github.com/hyphacoop/staticpub.distributed.press/pull/1/commits/4b85e21258f4193edf0216cc1a7c48591396e12d

**Pull request merged**